apps/topics/schema.py

```diff
@@ -0,0 +1,10 @@
+import graphene
+
+import apps.topics.models.schema as topics_schema
```

Comment:
nitpick: I'm not exactly sure why, but I've read that `from ... import ...` syntax is preferred. In this case: 
`from apps.topics.models import schema as topics_schema`